src/NUnitFramework/framework/Internal/Tests/TestSuite.cs

```diff
@@ -179,6 +179,16 @@ public void Add(Test test)
             tests.Add(test);
         }
 
+        /// <summary>
+        /// Creates a filtered copy of the test suite
+        /// </summary>
+        /// <param name="filter">Filter to apply</param>
+        /// <returns></returns>
```

Comment:
No `<returns>`, please. (https://github.com/nunit/docs/wiki/Best-practices-for-XML-documentation#returns-and-value)

```suggestion
```